src/NISysServer.py

```diff
@@ -20,6 +20,13 @@
0' type='6'>nisyscfg</Property><Property tag='101E001' type='6'>nisysmgmt</Property><Property tag='101F000' type='6'>%s</Property><Property tag='101F001' type='6'/><Property tag='1020000' type='1'>0</Property><Property tag='1022000' type='5'>{00000000-0000-0000-0000-000000000000}</Property><Property tag='1022001' type='5'>{DF2FE68E-F8DC-41AE-803C-75BD42785709}</Property><Property tag='1024000' type='2'>1</Property><Property tag='1028000' type='1'>0</Property><Property tag='102A000' type='2'>1000</Property><Property tag='102A001' type='2'>-10000</Property><Property tag='102F000' type='6'>8.8.0d39</Property><Property tag='1033000' type='6'>08:00:27:4B:05:76</Property><Property tag='1037000' type='3'>1376256</Property><Property tag='1037001' type='3'>1376256</Property><Property tag='1038000' type='1'>1</Property><Property tag='1039000' type='2'>1</Property><Property tag='103A000' type='3'>4</Property><Property tag='103C000' type='6'>cRIO</Property><Property tag='103D000' type='6'/><Property tag='104A000' type='1'>1</Property><Property tag='104B000' type='6'>*.cfg</Property><Property tag='104C000' type='2'>0</Property><Property tag='104E000' type='6'>NI-Linux x64</Property><Property tag='104F000' type='6'>4.14.146-rt67</Property><Property tag='1050000' type='6'>NI Linux Real-Time x64 4.14.146-rt67</Property><Property tag='1051000' type='7'>A9775000 B6165EA6 DD163B4A 0</Property><Property tag='1052000' type='1'>0</Property><Property tag='1053000' type='6'>Running</Property><Property tag='1054000' type='1'>0</Property><Property tag='1054001' type='1'>0</Property><Property tag='1058000' type='2'>1</Property><Property tag='1075000' type='6'/><Property tag='1083000' type='6'>Intel(R) Xeon(R) CPU E5-1650 v3 @ 3.50GHz</Property><Property tag='1084000' type='2'>2</Property><Property tag='10B2000' type='2'>1</Property><Property tag='5105000' type='1'>0</Property><Property tag='D11D000' type='6'>en</Property><Property tag='D11E000' type='6'>en</Property><Property tag='D120000' type='6'>CUT0</Property><Property tag='D122000' type='4'>10047024.000000</Property><Property tag='D123000' type='4'>9276488.000000</Property><Property tag='D129000' type='1'>1</Property><Property tag='D12B000' type='2'>0</Property><Property tag='D14E000' type='6'>UTC</Property><Property tag='D159000' type='6'/><Property tag='D15A000' type='6'/><Property tag='D15B000' type='6'/><Property tag='D15C000' type='6'/><Property tag='D15D000' type='1'>1</Property><Property tag='D15E000' type='1'>0</Property><Property tag='D180000' type='6'>8.8.0d39</Property><Property tag='D181000' type='6'>grub</Property><Property tag='13000000' type='1'>1</Property><Property tag='14000000' type='1'>1</Property><Property tag='14002000' type='1'>0</Property><Property tag='14004000' type='1'>0</Property><Property tag='1800B000' type='6'/><Property tag='1800C000' type='2'>0</Property><Property tag='1800D000' type='6'>VirtualBox--SN-0--MAC-08-00-27-4B-05-76</Property></PropertyBag></PropertyBags></NISysAPI_Results>"
```

Comment:
I don't think you should be hard-coding this response. It includes the IP address and even worse, the MAC address `08:00:27:4B:05:76`. The previous one didn't do this, by comparison. Please ensure you're returning only the properties you really need to.

Reply:
I fill in the ip address and hostname at run-time (search for the %s in the string constant), but you're correct that there's still things like gateway IPs, MAC address and other identifiable info in there.  I'll blank out the identifiable info since it seems as if the Reboot functionality only requires IP and hostname.